dynamic_rest/viewsets.py

```diff
@@ -338,6 +338,71 @@ def list_related(self, request, pk=None, field_name=None):
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
 
     ENABLE_BULK_PARTIAL_CREATION = settings.ENABLE_BULK_PARTIAL_CREATION
+    ENABLE_BULK_UPDATE = settings.ENABLE_BULK_UPDATE
+
+    def get_object(self):
+        lookup_url_kwarg = self.lookup_url_kwarg or self.lookup_field
+
+        if lookup_url_kwarg in self.kwargs:
+            return super(DynamicModelViewSet, self).get_object()
+
+        # If the lookup_url_kwarg is not present
```

Comment:
What is the purpose of this override? Is this to allow `PATCH` to be called without an ID?